gn3/api/async_commands.py

```diff
@@ -0,0 +1,21 @@
+"""Endpoints and functions concerning commands run in external processes."""
+import json
+
+import redis
+from flask import jsonify, Blueprint
+
+async_commands = Blueprint("async_commands", __name__)
+
+@async_commands.route("/state/<command_id>")
+def command_state(command_id):
+    with redis.Redis() as rconn:
```

Comment:
**nitpick:** With redis you can pass in a "decoded_responses" argument so that you save yourself the manual work of decoding things later.

```suggestion
    with redis.Redis(decode_responses=True) as rconn:
```

Reply:
TIL